tree.scss

```diff
@@ -66,31 +66,6 @@ $timing-fn: ease-out;
   width: 100%;
   height: 100%;
 
-  &.ie-trident {
-    /*
-     * In IE11 (oldest IE version supported by the tree), there's a bug
-     * and incoming nodes don't transition properly if the opacity isn't 1.
-     * So this disables the fade-in effect in IE11, so the translate transformation
-     * looks good.
-     */
-    .tree {
-      &.tree-overflow {
-        overflow-y: scroll;
-      }
```

Comment:
Why is this removed? overflow-y on the tree is for forcing IE11 to have scrollbars if this `tree-overflow` is set. That should be unrelated to this issue.